src/components/Header.html

```diff
@@ -22,15 +21,15 @@ <h4><a href="./HomePage.html" target="iframe-main-container">Vision Quiz</a></h4
       </form>
       
 <div class="iconsContainer flex-evenly">
-<a class="headerAnchorTag flex-column-center icons-btn-hover buttonHoverShadow" href="#"  target="#">  <i class="material-icons material-icons-text">person </i><span class="icon-inner-txt">Account</span> </a>
+<a class="headerAnchorTag flex-column-center icons-btn-hover buttonHoverShadow">  <i class="material-icons material-icons-text">person </i><span class="icon-inner-txt">Account</span> </a>
```

Comment:
You are still using camelcase in classes

Reply:
some classes I defined in CL I use Some properties from it. I'll Change it after submission